src/contracts/bodhi_token.js

```diff
@@ -1,8 +1,6 @@
 import Config from '../../config/config';
 import Contracts from '../../config/contracts';
-
-const Qweb3 = require('qweb3');
-const qweb3 = new Qweb3(Config.QTUM_RPC_ADDRESS);
+const Contract = require('qweb3/src/contract');
 
```

Comment:
Out of curiosity, the reason you stripped Contract out of qweb3 is that we don't need to call new twice?

Reply:
1. The npm module import couldn't find the Contract class.
2. qweb3 functionality didnt need to be in the Contract instance.
3. extra step for creating qweb3 instance first.